### DISCUSSION TITLE
GrandOrgue becomes unresponsive after attempting to change audio output device

### DISCUSSION THREAD

**Discussion by ashervahalik** [GrandOrgue/grandorgue#2077] [Help]
I booted up GrandOrgue as normal, and after completing my MIDI setup, I went to audio setup to change the audio source from my MacBook's built-in speakers to my wired headphones. I went to Settings...>Audio>Change and selected the option labeled, "PortAudio: coreaudio: External Headphones". This is something I do all the time, but this time around, it decided not to play any audio, though "External Headphones" is literally the system's name for the headphones, and now when I try to navigate to Settings...>Audio>Change, as soon as I click on "Change", GrandOrgue becomes unresponsive and requires a Force Quit.

I would at least like a temporary workaround (maybe a simple modification to a plist or xml?) for now if not something more permanent. I can't hear the audio from any of my organs or switch to a different output. And yes, the selected output doesn't play anything either, though, as I said, it was literally the output I selected.

**Comment by oleg68** [comment]
1. What version of Grandorgue are you using?
2. Have you tried to restart GrandOrgue? Does the sound work before changing the audio device?

**Reply by ashervahalik** [reply]
Sorry it took me so long to reply.

1. I was using the previous version of GrandOrgue, v3.15.3-1. After a bit of testing, this issue has not occured again on this particular version.
2. Restarting GrandOrgue did not help. And no, originally absolutely no sound would output at all. After I finagled with the settings a little bit, though, I fixed it, but I couldn't seem to reproduce the issue again.
